io.openems.edge.controller.evcs/src/io/openems/edge/controller/evcs/EvcsController.java

```diff
@@ -126,7 +138,27 @@ protected void deactivate() {
 
 	@Override
 	public void run() throws OpenemsNamedException {
-		Evcs evcs = this.componentManager.getComponent(this.evcsId);
+		
+		Boolean isClusterd = Boolean.valueOf((this.evcs.isClustered().value().toString()));
```

Comment:
Why convert from Integer to String to Boolean? Why not use a boolean-Channel from the beginning?

Reply:
Always use the same variable names ("isClusterd" vs. "isClustered") for the same information